python-flask-server/static/src/bot.js

```diff
@@ -42,6 +42,8 @@ function displayMessage(text, user) {
         // Set chat bubble color and position based on the user parameter
         if (user === watson) {
             bubble.className = 'bot_message';  // Bot text formatting
+            text = text.replace(/`/g, '')
```

Comment:
I figured out why these 2 lines were added.  Wouldn't it be better to have a formatter function and replace `something` w/ <code>something</code> and *another thing* with <strong>another thing</strong>?  Not sure the UI allows it, but I'm guessing it does (or could).

Probably something to consider trying later or next.

Reply:
I am revamping the UI similar to assistant slots. this js file might go away.